.gitignore

```diff
@@ -42,4 +42,4 @@ cmake-build-*
 default.profdata
 default.profraw
 
-tpcds_cached_tables/
```

Comment:
We should not remove that. However, I am not sure why this is part of the master and `tpch_cached_tables` is not.